testing/scripts/kind_test_all.sh

```diff
@@ -125,18 +125,32 @@ if [[ ${KIND_EXIT_VALUE} -eq 0 ]]; then
             echo "SKIPPING PREPACKAGED IMAGE BUILD..."
         fi
 
+        echo "Files changed in alibi explain folder:"
+        git --no-pager diff --exit-code --name-only origin/master ../../components/alibi-explain-server/
+        ALIBI_MODIFIED=$?
+        if [[ $ALIBI_MODIFIED -gt 0 ]]; then
```

Comment:
You should change this to ALIBI_EXPLAIN_MODIFIED and same below so they are separate (and then add them to the values at the bottom)

Reply:
done